app/models/piece.rb

```diff
@@ -100,4 +100,22 @@ def is_obstructed_diagonally(piece_destination)
       end 
     end 
   end 
+
+  def move_to!(x_new, y_new)
+    piece.position_x = x_current
+    piece.position_y = y_current
+    position_x = x_destination
+    position_y = y_destination
```

Comment:
Not sure that I understand the purpose of the first 4 lines of the method.. 🤔

Reply:
I wrote those to help keep track of which current and destination x and y coordinates I was using where. I thought it made it a little more readable.